doc/jemalloc.xml.in

```diff
@@ -1185,6 +1185,40 @@ mallctl("arena." STRINGIFY(MALLCTL_ARENAS_ALL) ".decay",
         enabled.  The default is <quote></quote>.</para></listitem>
       </varlistentry>
 
+      <varlistentry id="opt.stats_interval">
+        <term>
+          <mallctl>opt.stats_interval</mallctl>
+          (<type>uint64_t</type>)
+          <literal>r-</literal>
+        </term>
+        <listitem><para>Average interval between statistics outputs, as measured
+        in bytes of allocation activity.  The actual interval may be sporadic
+        because decentralized event counters are used to avoid synchronization
+        bottlenecks.  When triggered, the
+        <function>malloc_stats_print()</function> function is called.  <link
+        linkend="opt.stats_interval_opts"><mallctl>opt.stats_interval_opts</mallctl></link>
+        can be combined to specify output options.  The default value is 0,
```

Comment:
It seems like `-1` is a better "disabled" option than 0; 0 is in the domain of "possible values for allocator byte counts" while `-1` isn't (and has the -1 == infinity connection).

Reply:
I was thinking that 0 would mean the same as 1 as in bytes allocated. But you convinced me; will change to -1.